mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -288,6 +290,7 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 		ARG 1 nbt
 			COMMENT the NBT compound to write to
 	METHOD m_qxfxfsuy getItem ()Lnet/minecraft/unmapped/C_vorddnax;
+		COMMENT Called when another {@link ItemStack} is placed on this stack.
```

Comment:
This comment seems misplaced
```suggestion
```

Reply:
So that's where my disappearing javadoc went